### PR TITLE
Feat board player

### LINKED ISSUE #11: Add api endpoint to get all registered boards
and update client

### LINKED ISSUE #18: Remove game from `server.games` when ended

### PR DESCRIPTION
**Pull request by ggerbaud**
fix #11 
fix #18

### CODE REVIEW

File: server/server.go [hunk 1]
```diff
@@ -231,3 +290,34 @@ func (server *Server) getStylesHandler(c *gin.Context) {
 
 	c.JSON(http.StatusOK, gin.H{"styles": common.GsStyles})
 }
+
+func (server *Server) findBoard(board string) bool {
```
Comment (erwannT) [inline server/server.go]: findBoard is not correct for me because the board is not returned. Method verify only if board exist.

Comment (ggerbaud) [inline server/server.go]: Ok, it's just a naming issue so ?

Comment (erwannT) [inline server/server.go]: Yes

File: server/server.go [hunk 2]
```diff
@@ -65,32 +73,67 @@ func (server *Server) Start() {
...
+		c.JSON(http.StatusNotFound, nil)
+		return
+	}
+
 	log.WithFields(log.Fields{"gameID": gameID}).Info("flushing game w/ id")
 	wsHandler := websocket.Handler(server.hubs[gameID].handle)
 	wsHandler.ServeHTTP(c.Writer, c.Request)
 }
 
 func (server *Server) listeGamesHandler(c *gin.Context) {
```
Comment (mathieu-pousse) [inline server/server.go]: hey dude, this is franglais !

**Comment by mathieu-pousse** [comment]
lgtm

**Comment by erwannT** [comment]
LGTM

**Pull request merged**